lib-utilities/config/odimra_config.json

```diff
@@ -1,23 +1,23 @@
 {
-	"RootServiceUUID": "",
-	"LocalhostFQDN": "",
-	"SearchAndFilterSchemaPath": "",
-	"RegistryStorePath": "",
+	"RootServiceUUID": "ac2f7880-9d8a-11ed-a8fc-0242ac120002",
+	"LocalhostFQDN": "deviceManager",
+	"SearchAndFilterSchemaPath": "/etc/deviceManager/configs/schema.json",
+	"RegistryStorePath": "/etc/deviceManager/registryStore/",
 	"KeyCertConf": {
-		"RootCACertificatePath": "",
-		"RPCPrivateKeyPath": "",
-		"RPCCertificatePath": "",
-		"RSAPublicKeyPath": "",
-		"RSAPrivateKeyPath": ""
+		"RootCACertificatePath": "/etc/deviceManager/certs/rootCA.crt",
```

Comment:
Path for certs and keys is different than in /src/config/config.yml, are they different certs? If not, can we set the same path?

Reply:
Please note that `config.yml` was updated too to match paths between two configs.